runtime/service/src/main/java/org/apache/polaris/service/reporting/MetricsReporter.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.polaris.service.reporting;
+
+import org.apache.iceberg.rest.requests.ReportMetricsRequest;
+
+public interface MetricsReporter {
```

Comment:
Iceberg already has a `MetricsReporter` and a `LoggingMetricsReporter`. I suppose you created new ones because you wanted to pass the warehouse and namespace names as well?